nixos/modules/security/pam.nix

```diff
@@ -771,6 +771,7 @@ let
 
         password = autoOrderRules [
           { name = "systemd_home"; enable = config.services.homed.enable; control = "sufficient"; modulePath = "${config.systemd.package}/lib/security/pam_systemd_home.so"; }
+          { name = "pwquality"; enable = config.security.pam.pwquality.enable; control = "required"; modulePath = "${lib.getLib config.security.pam.pwquality.package}/lib/security/pam_pwquality.so"; }
```

Comment:
i'm not sure if this should be `requisite` `sufficient` or `required` tbh.
the docs (https://github.com/libpwquality/libpwquality/blob/master/doc/man/pam_pwquality.8.pod#examples) show `required` for this and `pam_unix` but this blogpost (https://deer-run.com/users/hal/linux_passwords_pam.html)  uses `requisite` for this and `sufficient` like we do for `pam_unix`

Reply:
I also noticed `pam_unix` seems to need `use_authtok` so I add that if pam_pwquality is enabled

Reply:
It should be either `requisite` or `required`. I would suggest `requisite` to be safe, since that will prevent any later modules from running at all. I don't know whether it makes a difference in the `password` flow in practice.